debian/clearwater-socket-factory.prerm

```diff
@@ -52,6 +52,12 @@ set -e
 
 case "$1" in
     remove|upgrade|deconfigure)
+        # Stop and deconfigure the management and signaling upstart jobs
+        stop clearwater-socket-factory-mgmt > /dev/null 2>&1 || true
+        stop clearwater-socket-factory-sig > /dev/null 2>&1 || true
```

Comment:
Will the package manager know to stop these two services (when the package itself is clearwater-socket-factory, not clearwater-socket-factory-mgmt or clearwater-socket-factory-sig)?